lib/companies_house/client.rb

```diff
@@ -49,6 +49,14 @@ def officers(id)
       items
     end
 
+    def company_search(query, limit)
```

Comment:
Should we have a sensible default limit? Or maybe make this an optional parameter, and don't send a limit if one isn't specified? (assuming the Companies House API has a default)